regression-tests.api/test_cryptokeys.py

```diff
@@ -0,0 +1,269 @@
+import subprocess
+import json
+import unittest
+
+from test_helper import ApiTestCase, is_auth
+
+@unittest.skipIf(not is_auth(), "Not applicable")
+class Cryptokeys(ApiTestCase):
+    zone = "cryptokeyzone"
```

Comment:
Could you state the reasons for this? We delete the created zone during "tear down".